extensions/wikia/PhalanxII/hooks/PhalanxContentBlock.class.php

```diff
@@ -207,17 +207,17 @@ static public function editContent( $textbox, &$error_msg, $phalanxModel = null,
 	 *
 	 * hook
 	 */
-	static public function checkContent( $textbox, &$msg ) {
+	static public function checkContent( $textbox, &$msg, $displayBlock = false ) {
 		wfProfileIn( __METHOD__ );
 
 		$title = RequestContext::getMain()->getTitle();
 
 		$phalanxModel = new PhalanxContentModel( $title );
 
 		$ret = PhalanxContentBlock::editContent( $textbox, $msg, $phalanxModel );
-		
-		if ( $ret === false ) {
-			$msg = $phalanxModel->textBlock();
```

Comment:
OK, I rechecked. I was going through this spaghetti code while having a bit of fever, but I think I've done it right—please correct me if I'm wrong. Here's what happens:
* on line above (`217`) script is calling `$ret = PhalanxContentBlock::editContent( $textbox, $msg, $phalanxModel );`,
* `$msg` is passed there by reference,
* `PhalanxContentBlock::editContent` [here (line `180`)](https://github.com/Wikia/app/blob/1094eb877751564535a0f27ac7b9a78306daec1c/extensions/wikia/PhalanxII/hooks/PhalanxContentBlock.class.php#L197) sets the message in the same way (`$error_msg = $phalanxModel->contentBlock();`),
* therefore setting `$msg = $phalanxModel->textBlock();` here (on line `ex 220`) was redundant.

Is it OK?